src/overlays/actors/ovl_En_M_Thunder/z_en_m_thunder.c

```diff
@@ -29,39 +39,533 @@ const ActorInit En_M_Thunder_InitVars = {
     (ActorFunc)EnMThunder_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_808B7120 = {
-    { COLTYPE_NONE, AT_ON | AT_TYPE_PLAYER, AC_NONE, OC1_NONE, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK2, { 0x01000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NONE, BUMP_ON, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_ON | AT_TYPE_PLAYER,
+        AC_NONE,
+        OC1_NONE,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK2,
+        { 0x01000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_NONE,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 200, 200, 0, { 0, 0, 0 } },
 };
 
-#endif
+static u8 sDamages[] = {
+    /* Regular    */ 1, 2, 3, 4,
+    /* Great Spin */ 1, 2, 3, 4,
+};
+
+static u16 sChargingSfxIds[] = {
+    /* ENMTHUNDER_SUBTYPE_SPIN_GREAT        */ NA_SE_IT_ROLLING_CUT_LV2,
+    /* ENMTHUNDER_SUBTYPE_SPIN_REGULAR      */ NA_SE_IT_ROLLING_CUT_LV1,
+    /* ENMTHUNDER_SUBTYPE_SWORDBEAM_GREAT   */ NA_SE_IT_ROLLING_CUT_LV2,
+    /* ENMTHUNDER_SUBTYPE_SWORDBEAM_REGULAR */ NA_SE_IT_ROLLING_CUT_LV1,
+};
```

Comment:
```suggestion
static u8 sDamages[] = {
    1, 2, 3, 4, // Regular    
    1, 2, 3, 4, // Great Spin
};

static u16 sChargingSfxIds[] = {
    NA_SE_IT_ROLLING_CUT_LV2, // ENMTHUNDER_SUBTYPE_SPIN_GREAT
    NA_SE_IT_ROLLING_CUT_LV1, // ENMTHUNDER_SUBTYPE_SPIN_REGULAR
    NA_SE_IT_ROLLING_CUT_LV2, // ENMTHUNDER_SUBTYPE_SWORDBEAM_GREAT
    NA_SE_IT_ROLLING_CUT_LV1, // ENMTHUNDER_SUBTYPE_SWORDBEAM_REGULAR
};
```
I think we prefer comments on the right, but I'm not sure if that's down to personal preference or not. I could also see the `sDamages` not formatting nicely with that

Reply:
I feel I've seen both (I don't really have a preference). If we do have a preference I don't mind changing them at all (formatting isn't an issue).

Reply:
I think the latter is easier to read in this case.